openapi.yaml

```diff
@@ -10317,6 +10734,30 @@ components:
           - application/vnd.whispir.contact-v1+json
           - application/vnd.whispir.contact-v1+xml
       description: Application specific mime-type.
+    Key-Content-Type:
```

Comment:
Per the previous comment on vendor-specific MIME type deprecation, we can remove this schema.

Reply:
I updated it to `Unified-Content-Type` with `application/json` so that it can be reused and avoid duplication of schema. We do have `Content-Type` here which uses `application/vnd.whispir.message-v1+json` but since it's being used, I didn't touch it.